src/azure-cli-core/azure/cli/core/extension/operations.py

```diff
@@ -485,7 +485,10 @@ def list_versions(extension_name, index_url=None, cli_ctx=None):
 
 
 def reload_extension(extension_name, extension_module=None):
-    return reload_module(extension_module if extension_module else get_extension_modname(ext_name=extension_name))
+    try:
+        return reload_module(extension_module if extension_module else get_extension_modname(ext_name=extension_name))
+    except KeyError:
+        pass
```

Comment:
How about we put the `try except` clause in https://github.com/Azure/azure-cli/blob/222e501067c4cbee305176d89198fbff46d4238c/src/azure-cli/azure/cli/command_modules/interactive/custom.py#L27

`az interactive` should be responsible for non-existing extension, such as `ExtensionNotInstalledException`.